api/swagger.yaml

```diff
@@ -1610,6 +1610,13 @@ definitions:
       ID:
         type: "string"
       Version:
+        description: |
+          The version number of the node object. This is required to avoid conflicting writes.
+          For every node object, the version number is incremented each time it is updated. It is required
```

Comment:
Not quite; the version numbers currently are taken from the Raft index, so they are global rather than local sequence numbers. But I think we should keep the `Version` field opaque and not provide any guarantees about when or how `Version` changes.